src/explore-education-statistics-common/src/styles/_print.scss

```diff
@@ -0,0 +1,37 @@
+@mixin print-hide() {
+  @media print {
+    display: none;
+  }
+}
+
+.dfe-print-hidden {
+  @include print-hide();
+}
+
+@mixin print-unbreakable() {
+  page-break-inside: avoid;
+}
+.dfe-print-unbreakable {
```

Comment:
Nitpick but unbreakable sounds weird, `dfe-print-break-avoid` is probably more inline with the CSS property itself